monix-reactive/shared/src/test/scala/monix/reactive/internal/operators/MapConcatSuite.scala

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2014-2019 by The Monix Project Developers.
+ * See the project homepage at: https://monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.reactive.internal.operators
+
+import monix.execution.Ack
+import monix.execution.Ack.{Continue, Stop}
+import monix.execution.exceptions.DummyException
+import monix.reactive.{Observable, Observer}
+
+import scala.concurrent.duration.Duration.Zero
+import scala.concurrent.duration._
+import scala.concurrent.{Future, Promise}
+
+object MapConcatSuite extends BaseOperatorSuite {
```

Comment:
It feels like we could add property based test `mapConcat(f) <-> flatMap(x => fromIterable(f(x))`

I'm also curious about performance difference but I can check myself later :D

Reply:
Good idea.
i can take care of that next week if  you don't mind waiting

Reply:
Do you have any pointers on doing that with minitest ?
thanks

Reply:
Take a look at [ReduceSuite](https://github.com/monix/monix/blob/master/monix-reactive/shared/src/test/scala/monix/reactive/internal/operators/ReduceSuite.scala#L63)

Reply:
Thanks, just pushed a test, let me know if that makes sense for you ?

Reply:
I think the test looks great :)